src/setting.js

```diff
@@ -210,7 +210,10 @@ let defaults = {
   'window.height': 604,
   'window.minheight': 440,
   'window.minwidth': 526,
-  'window.width': 564
+  'window.width': 564,
+  'window.max': false,
+  'window.preWidth': undefined,
```

Comment:
Do we really need extra settings for that? Can we not just use `window.width` and `window.height`?